index.html

```diff
@@ -2249,6 +2244,43 @@ <h3>
           available underneath an icon).
         </p>
       </section>
+      <section>
+        <h3>
+          Launching a web application
+        </h3>
+        <p>
+          At the discretion of the operating system and/or user agent, run the
+          steps to [=launch a web application=] with a
+          [=Document/processed manifest=].
+        </p>
+        <p class="note">
+          This would typically take place when the user selects an [=installed=]
+          web app from an app launching UI surface e.g., a home screen, launcher
+          or start menu.
+        </p>
+        <p>
+          The steps to <dfn data-lt="launching a web application">launch a web
+          application</dfn> is given by the following algorithm. The algorithm
+          takes a [=Document/processed manifest=] |manifest:processed manifest|,
+          an optional |target URL:URL| and an optional
+          <a data-cite="html#post-resource">POST resource</a> |POST resource|.
+        </p>
+        <ol class="algorithm">
+          <li>If |target URL| was not given, set |target URL| to [=start URL=].
+          </li>
+          <li>Let |traversable| be the result of
+              <a data-cite="html#create-a-fresh-top-level-traversable">
+              creating a fresh top-level traversable</a> with |target URL|
+              and |POST resource| (or null if |POST resource| was not given).
+          </li>
+          <li>Let |browsing context| be the |traversable|'s
+              <a data-cite="html#nav-bc">active browsing context</a>.
+          </li>
+          <li>[=Apply=] |manifest| to |browsing context| prior to the
+              |traversable| [=navigating=] to |target URL|.
+          </li>
+        </ol>
```

Comment:
Could you sneak in a "return _browsing context_" at the end here? (And say that it returns a browsing context.)

The return value will be ignored, for now, but you'll be able to make use of it in `web-app-launch` and it's not unreasonable for this algorithm to return the browsing context it creates.

Reply:
The replacement algorithm doesn't return anything either, no call site makes use of the browsing context.

Reply:
Added return, it's free and could be useful for a call site one day.